swagger.yaml

```diff
@@ -3091,6 +3094,10 @@ definitions:
         x-go-custom-tag: gorm:"type:text"
         type: string
         description: The configured NTP sources on the host.
+      disks_speed:
+        x-go-custom-tag: gorm:"type:text"
+        type: string
+        description: The result of disk performance executed prior to beginning installation.
```

Comment:
Is this some encoding of the speed per disk?  Is this the best way to store it?
@filanov thoughts?

Reply:
I was hoping for something more like this, whether it's stored in a column or table:
disk_info {
  [
    id: <id>
    speed_ms: 
    role: 
  ],
 ...
}
  ...

Reply:
Can try to do it as postgres json column.  Not sure I will be able to define the scheme here.

Reply:
Since we are using same struct to access db and for json serializing (swagger), it will be hard to use json column

Reply:
I'm not sure how to solve it. 
Maybe we need to refactor inventory into multiple tables? network/disks etc.

Reply:
refactoring into multiple tables is not relevant right now

@ori-amizur Why not add these fields (that @avishayt  mentioned ), in the swagger scheme?

Reply:
@filanov refactoring the inventory DB into multiple tables is not really an option right now, is it?

Reply:
No it's not, but if we continue to delay it we will never do it.

Reply:
I think it is not in the scope of this task.  We can do it in a different epic.